actix-settings/src/error.rs

```diff
@@ -28,28 +28,28 @@ pub enum Error {
     },
 
     /// I/O error.
-    #[display(fmt = "")]
+    #[display("I/O error: {_0}")]
```

Comment:
This is a functional change; I guess this is what is desired

Reply:
yep good spot, this is fine